shared-libs/constants/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@medic/constants",
+  "version": "1.0.0",
+  "description": "Shared constants for CHT-Core",
+  "main": "src/index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "nyc --nycrcPath='../nyc.config.js' mocha ./test"
+  },
```

Comment:
Is this "scripts" section needed or necessary? `shared-libs/constants/index.js` is just a grouping of constants and there are no direct tests for them.

Reply:
Thanks. I've removed the "directories" and "scripts" keys from the JSON in this file